src/scheduler/fifo.c

```diff
@@ -2452,6 +2397,8 @@ sched_settings_frm_svr(struct batch_status *status)
 				COPY_ATTR_VALUE(tmp_log_dir, attr->value);
 			} else if (!strcmp(attr->name, ATTR_partition)) {
 				COPY_ATTR_VALUE(tmp_partitions, attr->value);
+			} else if (!strcmp(attr->name, ATTR_comment)) {
+				COPY_ATTR_VALUE(tmp_comment, attr->value);
```

Comment:
This part of code is required to clear the comment of a corresponding scheduler at the server in the scenarios like the following.
1. Admin tries to set sched_log to a directory which does not exist for example. Then the scheduler comment is updated with the comment given below.
Qmgr: s sched sc1 sched_log = /tmp/a
Qmgr: l sc sc1
Sched sc1
    sched_host = stblr3
    pbs_version = 17.1.0
    sched_cycle_length = 00:20:00
    sched_port = 15050
    partition = p1
    sched_priv = /var/spool/pbs/suresh_priv_sc1
    sched_log = /tmp/a
    scheduling = False
    scheduler_iteration = 600
    comment = Unable to change the sched_logs directory
    state = idle
2. When admin tries to set the correct value for sched_log again then we need to clear the above comment.  
We need to clear the comment only if it is set earlier otherwise every time the user configures scheduler object correctly we need to clear the comment.